tango/executor.py

```diff
@@ -46,9 +51,11 @@ def __init__(
         self,
         workspace: Workspace,
         include_package: Optional[Sequence[str]] = None,
+        parallelism: Optional[int] = None,
     ) -> None:
         self.workspace = workspace
         self.include_package = include_package
+        self.parallelism = parallelism
```

Comment:
We should add a log statement here if `parallelism` is anything other than `None`, essentially saying what the note in the docstring says.

Reply:
Hmm that's a good point, but if we logged from `__init__()` we'd see that log statement from every subclass as well since the subclasses call `super().__init__()`. But I could put a warning/log statement in `execute_step_graph()` instead.